.travis.yml

```diff
@@ -4,6 +4,7 @@ sudo: false
 
 python:
   - '2.7'
+  - '3.6'
```

Comment:
Because this creates a matrix build, we need to add an additional condition to the `on:` configuration under `deploy:` for Python 3.6 so that only that build publishes to PyPi:

```yml
  on:
    tags: true
    python: "3.6"
    distributions: sdist bdist_wheel
    repo: azavea/python-omgeo
```